connectors/src/connectors/confluence/temporal/activities.ts

```diff
@@ -620,6 +620,9 @@ export async function confluenceGetReportPersonalActionActivity(
   if (connector.isAuthTokenRevoked) {
     return false;
   }
+  if (connector.errorType !== null) {
+    return false;
```

Comment:
The connector is errored with `third_party_internal_error`, I could specify it here instead of for all errors but I'm guessing if it's errored we don't want to process it no matter what the error was set. How does that sound?

Reply:
🤔

Reply:
I guess we only have this error alongside `third_party_internal_error` for now, right?

Reply:
we can set custom errors when using the command to error a connector right?

Reply:
updated to not process only if isThirdPartyInternalError or pausedAt which was not handled

Reply:
If connector is paused IMHO. we should still report data to Atlassian.

Reply:
Ok I added a commit!